### PR TITLE
tidb: add some privilege columns to mysql.db

### PR DESCRIPTION
**Pull request by tiancaiamao**
Add those columns:

```
Create_tmp_table_priv
Lock_tables_priv
Create_view_priv
Show_view_priv
Create_routine_priv
Alter_routine_priv
Execute_priv
Event_priv
Trigger_priv
```

Last time, I just add them to `mysql.user` and forget the `mysql.db`.
Now I've test the query in https://github.com/pingcap/tidb/issues/3323

### CODE REVIEW

File: bootstrap.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ const (
 	// CreateDBPrivTable is the SQL statement creates DB scope privilege table in system db.
 	CreateDBPrivTable = `CREATE TABLE if not exists mysql.db (
 		Host			CHAR(60),
-		DB				CHAR(64),
+		DB			CHAR(64),
```
Comment (hanfei1991) [inline bootstrap.go]: Why change this

Comment (tiancaiamao) [inline bootstrap.go]: Just make it aligned.

**Comment by hanfei1991** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**